styles/index.css

```diff
@@ -0,0 +1,102 @@
+html, body {
+  width: 100vw;
+  height: 100vh;
+  margin:auto;
+  display:grid;
+  grid-template-rows: repeat(10,10%);
+}
+
+header {
+    text-align: center;
+    padding: 20px;
+    background-image: url("../assets/image/giphy.gif");
+    background-size: 100% 100%;
+    color: white;
+    font-size:40px;
+    grid-row-start:1;
+    grid-row-end: 5;     
+}
+
+h2 {
+    font-size: 30px;
+}
+
+
+nav {
+    overflow: hidden; 
+    background-color:salmon;
+    padding: 1.px;
```

Comment:
It looks like there might be a typo here that is causing problems with the rest of the styling?